Cython/Compiler/Nodes.py

```diff
@@ -5023,6 +5034,13 @@ def analyse_declarations(self, env):
         self.entry = env.declare_property(self.name, self.doc, self.pos)
         self.entry.scope.directives = env.directives
         self.body.analyse_declarations(self.entry.scope)
+        # XXX DO SOMETHING HERE???
+        if 0 and self.is_wrapper:
+            entry = self.body.stats[0].entry
+            entry.is_property = 1
+            entry.doc = self.doc
+            env.property_entries[-1] = entry
+            env.entries[self.name] = entry
```

Comment:
The entry from `env.declare_property` is not being overridden by the later entry from the body.stats[0].entry, even if I enable the code here

Reply:
What is the intention here?

Reply:
sorry, removing old cruft.